cli/spec/fixtures/kontena_v3.yml

```diff
@@ -1,5 +1,6 @@
 stack: user/stackname
 version: 0.1.1
+labels: ['fqdn=wordpress.com']
```

Comment:
I'd prefer the multi-line array representation:

```yaml
labels:
  - fqdn=wordpress.com
```

Reply:
maybe replace these test fqdn:s with something like `example.com` instead of some actual sites.

Reply:
yup, will replace with something test-alike...

Reply:
example.com is reserved for things like that by whoever runs this internet thing.

Reply:
✅ RESOLVED